Fax numbers for gsync

sampatbadhe opened the pull request:
Hello Team,

This PR contain following changes -
- returns fax number in response.
- returns google type same as displayed in google.
- fix failing specs

Please review and advice.

Thanks.

sampatbadhe committed:
Fix Specs
- Fix timing related specs.
- Unescape ampersands from parsed link nodes. - Fixes a couple failing specs related to missing parameters in pagination links we construct.
ref - https://github.com/ryanlchan/google-contacts/pull/3

sampatbadhe committed:
- Add fax numbers to allow google sync
- build same label for email address, phone numbers and address which present in google

sampatbadhe committed:
add attr_accessor for fax numbers

sampatbadhe committed:
spec for fax numbers

sampatbadhe committed:
remove comment

sampatbadhe committed:
get google label from rel attribute of phone numbers, email addresses and addresses

sampatbadhe committed:
revert changes for split google label by url

rohandaxini commented on lib/google-contacts/element.rb:
```diff
@@ -289,7 +309,10 @@ def update_groups(*group_links)
 
     private
-     # Evil ahead
+      def get_google_type(google_type)
```
@sampatbadhe should this method name be more meaningful? What is google_type ?

sampatbadhe replied:
hello @rohandaxini 
This returns the label name which google used for `phone number`, `address`, `email address`...

rohandaxini replied:
@sampatbadhe should it be `get_google_label_name` or anything else that is more relevant? Method name `get_google_type` is very confusing and its not clear what it returns or does. It just has comment `# Evil ahead`

sampatbadhe replied:
@rohandaxini sure I'll change method name to `get_google_label_name`.

`# Evil ahead` this comment was there previously. I have removed it in my changes. :stuck_out_tongue_winking_eye:

rohandaxini commented:
@sampatbadhe rest all is fine :+1:

sampatbadhe committed:
add relevant method name

sampatbadhe committed:
lock rake gem version to pass travis-ci build
- build was failing with this error - NoMethodError: undefined method `last_comment' for #<Rake::Application:0x007f0b59664390>

sampatbadhe commented:
Thanks @rohandaxini

rohanpujaris commented:
:+1:

titame commented:
:+1:

abhijitsinha commented:
Hi @rohandaxini - this PR is good to merge to master. But please **do not delete** the branch as this branch has been referenced in Gemfile and used in `production` branch.

rohandaxini commented:
Sure @abhijitsinha
